prioritizer/prioritizer.py

```diff
@@ -27,58 +29,17 @@
 # File paths
 OUTPUT_FILE = "new_output.json"
 
-def send_to_llm(prompt):
-    response = client.chat.completions.create(
-        model="gpt-4o",
-        messages=[
-            {
-                "role": "system",
-                "content": """
-            You are a climate action expert, tasked to prioritize climate actions for cities.
-             
-            These are the rules to use for prioritizing a climate action for a city:
-             
-            - Lower cost actions are better than higher cost actions.
-            - High emissions reductions are better than low emissions reductions.
-            - High risk reduction is better than low risk reduction.
-            - Actions that match the environment are better than those that don't.
-            - Actions that match the population are better than those that don't.
-            - Actions that take less time are better than those that take more time.
-            """,
-            },
-            {"role": "user", "content": prompt},
-        ],
-    )
-    return response.choices[0].message.content
 
 def read_city_inventory():
     # Load both CSV files into DataFrames
-    activities_df = pd.read_csv("CAP_data/activities_one_row_per_activity.csv")
-    gases_df = pd.read_csv("CAP_data/activities_one_row_per_gas.csv")
-    
-    # Merge the two DataFrames on the 'id' column
-    combined_df = activities_df.merge(gases_df, on='id', suffixes=('_activity', '_gas'), how='left')
-
-    # Process and clean numeric fields as needed
-    for field in ["co2eq", "co2eq_years", "activity_value", "co2eq-2", "gas_amount", "emissions_per_activity"]:
-        if field in combined_df.columns:
-            combined_df[field] = combined_df[field].replace(",", "", regex=True).astype(float)
-    
-    # Aggregating data by city (assuming all entries in the combined DataFrame belong to a single city)
-    city_data = {
-        "total_co2eq": combined_df.get("co2eq", pd.Series(0)).sum(),
-        "total_activity_value": combined_df.get("activity_value", pd.Series(0)).sum(),
-        "average_emissions_per_activity": combined_df.get("emissions_per_activity", pd.Series(0)).mean(),
-        "total_gas_amount": combined_df.get("gas_amount", pd.Series(0)).sum(),
-        # Add other aggregated fields as necessary
-    }
-    
-    # Wrapping in a list to keep compatibility with original function structure
-    return [city_data]
+    city_data_path = "CAP_data/city_data.json"
+    with open(city_data_path, "r", encoding="utf-8") as f:
+        city_data = json.load(f)
+    return city_data[0]
```

Comment:
`city_data[0]` loads only the first element of the json no? How do you load the other values of the other cities?

Reply:
i did it only for one city for now that's why I don't iterate there was no point of going for 5 where we didn' fully decide on so many things